RecoEgamma/EgammaIsolationAlgos/python/interestingEgammaIsoDetIdsSequence_cff.py

```diff
@@ -45,6 +45,26 @@
 interestingGedGamIsoDetIdEE.innerRadius = 0.0
 
 import RecoEgamma.EgammaIsolationAlgos.interestingGamIsoDetIdModule_cff
+interestingOotGamIsoDetIdEB = RecoEgamma.EgammaIsolationAlgos.interestingGamIsoDetIdModule_cff.interestingGamIsoDetId.clone()
```

Comment:
this is too verbose and too much copy-paste
```python
 interestingOotGamIsoDetIdEB = interestingGedGamIsoDetIdEB.clone(emObjectLabel = 'ootPhotons')
``` 
should do the same and fits on just one line.